src/innhold/innhold-metrics.tsx

```diff
@@ -65,7 +65,7 @@ function Metrics() {
     });
 
     const hotjarEksperiment = () => {
-        return POAGruppe === 'kss' && ukerRegistrert < 4;
+        return POAGruppe === 'kss' && (ukerRegistrert > 4 && ukerRegistrert < 13);
```

Comment:
`ukerRegistrert` teller antall heltallige uker brukeren har vært registrert. Det betyr at dag 1–6 viser `ukerRegistert` 0, dag 7–13 viser 1, osv.
`ukerRegistrert > 4` vil derfor telle _etter_ fire uker, og ikke f.o.m. uke fire. `ukerRegistrert < 13` vil også ta med alt helt til brukeren har vært registrert i tretten uker.

For å få «fra uke 4» til uke 12, så tror jeg `ukerRegistrert >= 4 && ukerRegistrert < 12` blir riktig. Da får vi også tettet hullet mellom `ukerRegistrert < 4` og `ukerRegistrert > 4` (som ikke tar høyde for `ukerRegistert == 4`)